package.json

```diff
@@ -8,10 +8,8 @@
     "url": "https://github.com/strange-developer/react-maybe"
   },
   "scripts": {
-    "test":
-      "mocha-webpack --c --webpack-config webpack.config.js \"test/**/*.test.js\"",
-    "test:watch":
-      "mocha-webpack --c -w --webpack-config webpack.config.js \"test/**/*.test.js\""
+    "test": "mocha-webpack --c --webpack-config webpack.config.js \"test/**/*.test.js\"",
```

Comment:
@strange-developer Should I add the linting in here too?

Reply:
I'm indifferent to the linting here. Prettier automatically does it on my side.

Reply:
Okay, awesome. I will just add in another line to be able to run the linting separately so it's not part of the test script but available to run if ever needed, that cool?